CMakeLists.txt

```diff
@@ -14,6 +14,22 @@ project(Generators LANGUAGES C CXX)
 # All Options should be defined in cmake/options.cmake This must be included before any other cmake file is included
 include(cmake/options.cmake)
 
+# Apply custom compiler flags for TRT_RTX builds to match CUDA preset
+if(USE_TRT_RTX AND WIN32)
```

Comment:
So far, we've kept our c and cxx flags minimal in the cmakelists.txt and tried to supply these flags as build needs in [cmake presets.](https://github.com/microsoft/onnxruntime-genai/blob/4fd29d4ad11bddea0d2107c258544f0be8d8e895/cmake/presets/CMakeWinConfigPresets.json#L23-L29)

Would you like to consider doing the same?